scripts/test/Muon/LoadWidgetPresenter_test.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+
+from Muon.GUI.Common.load_widget.load_presenter import LoadPresenter
+from Muon.GUI.Common.load_widget.load_view import LoadView
+from Muon.GUI.ElementalAnalysis.LoadWidget.load_model import LoadModel, CoLoadModel
+
+from Muon.GUI.Common import mock_widget
+
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+
+class LoadPresenterTest(unittest.TestCase):
+    def setUp(self):
+        self._qapp = mock_widget.mockQapp()
+
+        self._view = mock.create_autospec(LoadView)
+        self._load_model = mock.create_autospec(LoadModel)
+        self._co_model = mock.create_autospec(CoLoadModel)
+        self.presenter = LoadPresenter(
+            self._view, self._load_model, self._co_model)
+        self.view = self.presenter.view
+
+    def test_equalise_loaded_runs(self):
+        self.presenter.co_model.loaded_runs = 5
+        self.presenter.load_model.loaded_runs = 10
+        self.presenter.equalise_loaded_runs()
+        assert self.presenter.co_model.loaded_runs == 10
+
+    def test_update_models(self):
+        self.presenter.load_model = LoadModel()
+        self.presenter.co_model = CoLoadModel()
+        test_value = 10
+        self.presenter.update_models(test_value)
+        assert self.presenter.load_model.run == test_value
+        assert self.presenter.co_model.run == test_value
+
```

Comment:
add test for enable and disable buttons

Reply:
you can check that connects happen too

Reply:
if we test connects then we know signals and slots will trigger correctly, see https://github.com/mantidproject/mantid/pull/23150

Reply:
The connects you've tested are custom signals, whereas the ones here are methods on the various QWidgets (buttons etc.), so these should be tested already, no? I can add tests, but that'll mean creating an autospec for each 'widget', then mocking all the methods for each one

Reply:
we are not testing the signal, its that the signal connects to the correct slot. I am sure the signal works (and I have assumed this)

Reply:
I think I know what you're getting at - do you want me to test that the connect methods are *called* within the `on_load_clicked` etc., rather than testing that they actually connect?

Reply:
yes, so if we know the connect is called and with the correct arguments, then we can assume everything works